src/common/helpers.ts

```diff
@@ -64,6 +65,22 @@ export function date(date: number | string, format: string) {
   return dayjs(date).format(format);
 }
 
+export function useFormatTime() {
```

Comment:
Isn't this a duplicate of `useDateTime`?

Reply:
@beganovich Actually, it's not a duplicate because `useFormatTime` is dedicated only to formatting time values, while `useDateTime` formats the full `date + time`. However, since you mentioned it, I believe it's better to have it under `useDateTime` so we can make it more global. So, I've modified the `useDateTime` hook to accept time-only formatting. Let me know your thoughts.